autobuild/windows.ps1

```diff
@@ -0,0 +1,159 @@
+param(
```

Comment:
If we really want this to only be GitHub compatible, I'd add a comment here to say that this is a GitHub only script.

Maybe also create a README.md file in autobuild/

Reply:
> If we really want this to only be GitHub compatible, I'd add a comment here to say that this is a GitHub only script.

Good idea, done.

> Maybe also create a README.md file in autobuild/

Not sure. What would we add there? The fact that the directory is called autobuild/ should also be halfway obvious. Maybe we should move it to a more appropriate place altogether? Something like .github/autobuild/ similar to .github/action_scripts?

I also want to re-emphasize that the scripts already used Github-specific logic and this is nothing which is introduced as part of this PR. Therefore I want to limit the work in that direction to a minimum to get this PR merged quickly. I'm not opposed to further improvements afterwards, of course. :)

Reply:
I think a one line README.md so people not familiar with Github understand is a good idea.

Reply:
> .github/action_scripts

Yes. Probably that would be best:

./github/autobuild_scripts

Reply:
Ok, added a README and used .github/autobuild* now.